gpu_bdb/queries/q25/gpu_bdb_query_25_dask_sql.py

```diff
@@ -36,19 +37,27 @@
 from dask import delayed
 
 def get_clusters(client, ml_input_df):
+    import dask.dataframe as dd
     import dask_cudf
-
+    import cudf
+    import pandas as pd
+    
     ml_tasks = [
         delayed(train_clustering_model)(df, N_CLUSTERS, CLUSTER_ITERATIONS, N_ITER)
         for df in ml_input_df.to_delayed()
     ]
     results_dict = client.compute(*ml_tasks, sync=True)
 
     output = ml_input_df.index.to_frame().reset_index(drop=True)
-
-    labels_final = dask_cudf.from_cudf(
-        results_dict["cid_labels"], npartitions=output.npartitions
-    )
+    
```

Comment:
Any reason we can't use the same `get_clusters` for both the dask_cudf and dask-sql versions of the query like in query 20?